packages/website/ts/components/forms/subscribe_form.tsx

```diff
@@ -0,0 +1,121 @@
+import { colors } from '@0xproject/react-shared';
+import * as React from 'react';
+
+import { logUtils } from '@0xproject/utils';
+import { Button } from 'ts/components/ui/button';
+import { Container } from 'ts/components/ui/container';
+import { Input } from 'ts/components/ui/input';
+import { Text } from 'ts/components/ui/text';
+import { styled } from 'ts/style/theme';
+import { backendClient } from 'ts/utils/backend_client';
+
+export interface SubscribeFormProps {}
+
+export enum SubscribeFormStatus {
+    None,
+    Error,
+    Success,
+    Loading,
+}
+
+export interface SubscribeFormState {
+    emailText: string;
+    lastSubmittedEmail: string;
+    status: SubscribeFormStatus;
+}
+
+export class SubscribeForm extends React.Component<SubscribeFormProps, SubscribeFormState> {
+    public state = {
+        emailText: '',
+        lastSubmittedEmail: '',
+        status: SubscribeFormStatus.None,
+    };
+    public render(): React.ReactNode {
+        const formFontSize = '15px';
+        return (
+            <Container className="flex flex-column items-center justify-between md-mx2 sm-mx2">
```

Comment:
Mostly by accident. But I don't think it's a bad thing ... I do think this is a nice level of abstraction to work on (always be using components with a clear API instead of raw elements). 
```
<Container {...layout props...}>
    <Text {...text props...}>
        hey 
    <Text>
</Container>
```
Is easier to read / has clearer separation of concerns than
```
<div style={... styles for layout and for the text style...}>
    hey
</div>
```